frontend/scripts/react-components/tool/tool-layout/tool-layout.component.jsx

```diff
@@ -0,0 +1,234 @@
+        <div className="js-map-legend-choro c-map-legend-choro" />
+      </div>
+    </div>
+    <div className="js-map-attribution c-map-attribution">
+      {/* this is rendered by map.component */}
+    </div>
+  </div>
+);
+
+const renderSankeyError = () => (
+  <div className="js-sankey-error is-hidden">
+    <div className="veil -with-menu -below-nav" />
+    <div className="c-modal -below-nav">
+      <div className="content -auto-height">
+        The current selection produced no results. This may be due to data not being available for
+        the current configuration or due to an error in loading the data. Please change your
+        selection or reset the tool to its default settings.
+        <button className="c-button js-sankey-reset">reset</button>
+      </div>
+    </div>
+  </div>
+);
+
+const renderExpandButton = () => (
+  <div className="expand-button js-expand">
+    <svg className="icon icon-outside-link">
+      <use xmlnsXlink="http://www.w3.org/1999/xlink" xlinkHref="#icon-more-options" />
+    </svg>
+    <div className="c-node-menu">
+      <ul className="actions">
+        <li
+          className="expand-action js-expand-action is-hidden"
+          data-expand-text="EXPAND"
+          data-re-expand-text="RE-EXPAND"
+        >
+          <svg className="icon">
+            <use xlinkHref="#icon-more-options" />
+          </svg>
+        </li>
+        <li className="collapse-action js-collapse-action is-hidden">
+          COLLAPSE
+          <svg className="icon">
+            <use xlinkHref="#icon-more-options" />
+          </svg>
+        </li>
+        <li className="js-clear">CLEAR SELECTION</li>
+      </ul>
+    </div>
+  </div>
+);
+
+const renderSankey = () => (
+  <div className="c-sankey is-absolute js-sankey">
+    <div className="js-sankey-scroll-container sankey-scroll-container">
+      <svg className="js-sankey-canvas sankey">
+        <defs>
+          <pattern
+            id="isAggregatedPattern"
+            x="0"
+            y="0"
+            width="1"
+            height="3"
+            patternUnits="userSpaceOnUse"
+          >
+            <rect x="0" y="0" width="50" height="1" fill="#ddd" />
+            <rect x="0" y="1" width="50" height="2" fill="#fff" />
+          </pattern>
+        </defs>
+        <g className="sankey-container">
+          <g className="sankey-links" />
+          <g className="sankey-columns">
+            <g className="sankey-column">
+              <g className="sankey-nodes" />
+            </g>
+            <g className="sankey-column">
+              <g className="sankey-nodes" />
+            </g>
+            <g className="sankey-column">
+              <g className="sankey-nodes" />
+            </g>
+            <g className="sankey-column">
+              <g className="sankey-nodes" />
+            </g>
+          </g>
+        </g>
+      </svg>
+    </div>
+
+    {renderExpandButton()}
+  </div>
+);
+
+const ToolLayout = ({ store }) => {
```

Comment:
Could we call it `Tool`? in the rest of the pages the layout is called `${parentDirectory}.component.js`

Reply:
Actually, I'm going to move it directly to the react-components/tool directory. This is the main component and it should be at the root